presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java

```diff
@@ -317,6 +317,36 @@ public void testColumnComments()
         dropTable(session, "test_column_comments");
     }
 
+    @Test
+    public void testTableComments()
+    {
+        testTableCommentsForFormat(FileFormat.ORC);
```

Comment:
We don't need to test multiple formats here, as table comment is only in the metadata, not related to file formats.

Reply:
I changed the test to only test the ORC format.  I also added support and tests for COMMENT ON TABLE tablename IS (comment|null).